### PR TITLE
Lombok Library Update

### PR DESCRIPTION
**Pull request by jayandran-Sampath**
## Change list

Used Lombok Library to generate generic getter & setter for the classes.

ShortNote about Lombok : - 
Lombok can generate getters and setters for those object automatically by using Lombok annotations which helps in reducing the number of line & compilation time
 
## Types of changes

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [ ] No changes in production code.
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

Please provide more details about changes if it is necessary. If there are new features you can provide code samples which show the way they
work and possible use cases. Also you can create [gists](https://gist.github.com) with pasted java code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://help.github.com/categories/writing-on-github/)

### CODE REVIEW

**Comment by SrinivasanTarget** [comment]
@jayandran-Sampath Thanks for your PR. Reviewing it.

File: src/main/java/io/appium/java_client/AppiumCommandInfo.java [hunk 1]
```diff
@@ -16,12 +16,14 @@
...
 
+import lombok.AccessLevel;
+import lombok.Getter;
 import org.openqa.selenium.remote.CommandInfo;
 import org.openqa.selenium.remote.http.HttpMethod;
 
 public class AppiumCommandInfo extends CommandInfo {
-    private final String url;
-    private final HttpMethod method;
+    @Getter(AccessLevel.PUBLIC) private final String url;
```
Comment (SrinivasanTarget) [inline src/main/java/io/appium/java_client/AppiumCommandInfo.java]: `Public` need not to be set explicitly in Lombok AFAIK. But no harm

File: build.gradle [hunk 2]
```diff
@@ -29,26 +29,33 @@ apply plugin: 'com.github.johnrengelman.shadow'
...
     options.define compilerArgs: [
             '-encoding', 'utf-8'
     ]
 }
 
 dependencies {
+    compileOnly('org.projectlombok:lombok:1.18.8')
+    testCompileOnly('org.projectlombok:lombok:1.18.8')
+    annotationProcessor('org.projectlombok:lombok:1.18.8')
+    testAnnotationProcessor('org.projectlombok:lombok:1.18.8')
```
[the remark this answers is not in the document]
Comment (jayandran-Sampath) [reply build.gradle]: will remove both testCompile & testAnnotation. If its required to implement in lombok library in test classes we can add it later. thanks

File: src/main/java/io/appium/java_client/AppiumCommandInfo.java (hunk 1 above)
Comment (jayandran-Sampath) [reply src/main/java/io/appium/java_client/AppiumCommandInfo.java]: Since prior implementation was public method call, i have implemented the same.

**Pull request merged**